BOJ/solved.ac_class/Class02/9023.괄호/sAp00n.py

```diff
@@ -0,0 +1,28 @@
+from sys import stdin
+from collections import deque
+
+
+def checker(test_case):
+    que = deque()
+    for idx in range(len(test_case)):
+        if test_case[idx] == ')':
+            if '(' not in que:
+                return False
+            else:
+                que.remove('(')
```

Comment:
```suggestion
            que.remove('(')
```
return 다음에 나오는 else는 생략해주세요!